torch/linalg/__init__.py

```diff
@@ -453,7 +453,6 @@
 .. seealso::
 
         :func:`torch.linalg.eigvalsh` computes only the eigenvalues.
```

Comment:
Perhaps add a line explaining that this function does not have unicity / gradient problems?

Reply:
All gradients in PyTorch should be stable by default, I think we need to document only unstable and unexpected cases.

Reply:
I just wanted to stress the difference. Perhaps write it as:
> Unlike :func:\`torch.linalg.eigh\`, the gradients of this function are always stable
?

Reply:
Highlighting the difference(s) with the current operation sounds good. Maybe something as simple as "only the eigenvalues, but its gradients are always stable."?

I still don't like referring to gradients as "stable" but that's an issue for another time.

Reply:
I updated this section and I used the term "numerically stable". Which is a well-defined concept
https://nhigham.com/2020/08/04/what-is-numerical-stability/
https://en.wikipedia.org/wiki/Numerical_stability